src/Common/OvercommitTracker.cpp

```diff
@@ -23,6 +23,12 @@ void OvercommitTracker::setMaxWaitTime(UInt64 wait_time)
 
 bool OvercommitTracker::needToStopQuery(MemoryTracker * tracker)
 {
+    // NOTE: DO NOT CHANGE THE ORDER OF LOCKS
```

Comment:
```suggestion
    // NOTE: Do not change the order of locks
```
I don't like when comments yelling on me.